### PR TITLE
Update image target in Makefile

### PR DESCRIPTION
**Pull request by dmage**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,13 +8,16 @@ include $(addprefix ./vendor/github.com/openshift/library-go/alpha-build-machine
...
 # $0 - macro name
 # $1 - target suffix
-# $2 - Dockerfile path
-# $3 - context directory for image build
+# $2 - image ref
+# $3 - Dockerfile path
+# $4 - context directory for image build
 # It will generate target "image-$(1)" for builing the image an binding it as a prerequisite to target "images".
-$(call build-image,origin-$(GO_PACKAGE),./images/Dockerfile,.)
+$(call build-image,ocp-openshift-apiserver,$(IMAGE_REGISTRY)/ocp/4.2:openshift-apiserver,./images/Dockerfile,.)
```
Comment (tnozicka) [inline Makefile]: ./images/Dockerfile.rhel

**Comment by dmage** [comment]
Up, is there any way to build the api server without such changes?

/cc @sttts @mfojtik

**Comment by tnozicka** [comment]
/approve

the change sgtm but I can't test it, you need a hack like https://github.com/openshift/cluster-config-operator/pull/98/files#diff-f7c5b4068637e2def526f9bbc7200c4eR1

otherwise I get

**Comment by dmage** [comment]
```
$ go get -u github.com/openshift/imagebuilder/cmd/imagebuilder
$ imagebuilder -version
1.1-dev
$ make images
imagebuilder --allow-pull -t registry.svc.ci.openshift.org/ocp/4.3:openshift-apiserver -f ./images/Dockerfile.rhel .
--> FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.12 as builder
--> WORKDIR /go/src/github.com/openshift/openshift-apiserver
--> COPY . .
--> RUN make build --warn-undefined-variables
<REDACTED>
--> Done
$ ls -al vendor/github.com/coreos/etcd/Documentation/
ls: cannot access 'vendor/github.com/coreos/etcd/Documentation/': No such file or directory
```

@tnozicka what commit are you trying to apply this change to?

**Pull request closed without merging**